backends/bmv2/psa_switch/psaSwitch.h

```diff
@@ -187,6 +187,11 @@ class ParsePsaArchitecture : public Inspector {
     explicit ParsePsaArchitecture(PsaProgramStructure* structure) :
         structure(structure) { CHECK_NULL(structure); }
 
+    void modelError(const char* format, const IR::INode* node) {
+        ::error(format, node->getNode());
```

Comment:
should perhaps be `ErrorType::ERR_INVALID`?  Or perhaps a new ERR_ARCH_INTERNAL or some such